lib/Conversion/TorchToArith/TorchToArith.cpp

```diff
@@ -203,8 +203,8 @@ class ConvertTorchTensorLiteralOp
         if (auto intType = type.getElementType().dyn_cast<IntegerType>()) {
           Type builtinTensorElemTy =
               IntegerType::get(context, intType.getIntOrFloatBitWidth());
-          auto shapedType =
-              RankedTensorType::get(type.getShape(), builtinTensorElemTy);
+          auto shapedType = RankedTensorType::get(
+              makeShapeLLVMCompatible(type.getShape()), builtinTensorElemTy);
```

Comment:
This does not make sense, since `type` is already a RankedTensorType so it should already be LLVM compatible.